Controllers/EmployeeController.cs

```diff
@@ -16,12 +16,105 @@ namespace TodoApi.Controllers
     public class EmployeeController : ControllerBase
     {
         private readonly ApiDbContext _context;
+        private readonly string connectionString = "Server=myserver;Database=mydb;User Id=admin;Password=password;";
 
 
         public EmployeeController(ApiDbContext context)
```

Comment:
Avoid using `Thread.Sleep` in async methods as it blocks the thread; use `Task.Delay` instead.